wireless/bluetooth/bt_hcicore.c

```diff
@@ -1622,24 +1616,35 @@ void bt_driver_unregister(FAR const struct bt_driver_s *btdev)
 
 /* TODO: rename to bt_receive? */
```

Comment:
can yo do this rename? there's an asymmetry in the naming of functions in this file

Reply:
Done